scripts/windows-testing/parallel-executor.mjs

```diff
@@ -30,7 +30,6 @@ async function main() {
           PATH: process.env.PATH,
           E2E_SHARD_TOTAL: process.env.E2E_SHARD_TOTAL,
           E2E_SHARD_INDEX: process.env.E2E_SHARD_INDEX,
-          FORCE_COLOR: '3',
```

Comment:
Maybe `0` to forcefully disable it and have consistency between local and CI runs.